### ISSUE TITLE
Use scikit-learn StandardScaler for standardizing features

### ISSUE THREAD

**Issue by JWarmenhoven** [JWarmenhoven/ISLR-python#5]
Use sklearn.preprocessing.StandardScaler to scale training set features and apply same transform to the test set.
